sql/catalyst/src/main/scala/org/apache/spark/sql/types/dataTypes.scala

```diff
@@ -300,6 +301,8 @@ case object StringType extends NativeType with PrimitiveType {
    * The default size of a value of the StringType is 4096 bytes.
    */
   override def defaultSize: Int = 4096
+
+  override def toSimpleString = "string"
```

Comment:
Here to replace `TypeName` to `string` is ok.
But not all `typeName` equals `simple string`. 
i.e
LongType, the `typeName` of it is `long` but not `bigint`... etc...
> 
 def typeName: String = this.getClass.getSimpleName.stripSuffix("$").dropRight(4).toLowerCase